charts/openebs/openebs-nfs-kill-chaos/experiment.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Kill the OpenEBS NFS provisioner container and check if pods mounted with RWM modes.
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: openebs
+  name: openebs-nfs-kill-chaos
```

Comment:
openebs-nfs-provisioner-kill -- I see this used in our exp ansible logic. We need to keep these names consistent. Else there will be a mismatch with result resource naming.